### DISCUSSION TITLE
Some Ideas

### DISCUSSION THREAD

**Discussion by Voigtus** [EugenMayer/helm-charts#24] [Ideas]
Hai there, 

I was just scanning through the rundeck helm chart. 
And I saw some parts, where me and a friend saw some opportunities. 

Are you open for PR, or is this helm chart for a specific use case and therefore not open for PR?
And how would you like to get them? Issue into PR? Out of the blue PR? Discussion before PR?

Greetings,
Patrick

**Comment by EugenMayer** [comment]
Hello @Voigtus - thank you for considering to contribute :)

To your question, usually, i would advise to always create an issue or a discussion first, with the idea/intention you want to implement if it is a feature or an improvement. Have a short conversation about if the feature make sense/would be merged if implemented and maybe some essential things to consider during the implementation so it gets merged.

Usually, this process is relatively quick and mostly it is a simple 'sure go for it' or 'well it will not be part of this project due to ..'

For bugs, just be bold, a PR is better then anything else :)
